sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -244,10 +249,40 @@ def encode_message(self):
         return self.message.encode_message()
 
 
-class _BatchSendEventData(EventData):
-    def __init__(self, batch_event_data, partition_key=None):
-        self.message = BatchMessage(data=batch_event_data, multi_messages=False, properties=None)
+class EventDataBatch(object):
+    """
+    The EventDataBatch class is a holder of a batch of event date within max message size bytes.
+    Use ~azure.eventhub.Producer.create_batch method to create an EventDataBatch object.
+    Do not instantiate an EventDataBatch object directly.
+    """
+
+    log = logging.getLogger(__name__)
+
+    def __init__(self, max_size=None, partition_key=None):
+        self.max_size = max_size if max_size else constants.MAX_MESSAGE_LENGTH_BYTES
```

Comment:
This can be a simple or-statement:
`self.max_size = max_size or contants.MAX_MESSAGE_LENGTH_BYTES`

Reply:
thanks for the suggestion, code updated.